doc/api/util.md

```diff
@@ -126,6 +126,10 @@ util.format(1, 2, 3); // '1 2 3'
 
 ## util.inherits(constructor, superConstructor)
 
+_Note: usage of `util.inherits()` is discouraged. Please use ES6 `class` and
+`extends` to get language level inheritance support, including an unbroken
+inheritance chain._
```

Comment:
I'd rewrite this to explain in some detail what it does wrong and why it can't be fixed without breaking existing code.  Saying 'unbroken inheritance chain' probably isn't very elucidating unless you know how `util.inherits()` works, implementation-wise.